src/readers/morphologyASC.cpp

```diff
@@ -281,6 +281,12 @@ class NeurolucidaParser
                 }
                 return true;
             } else if (is_end_of_branch(id)) {
```

Comment:
I am really bad with the asc format.  Are we sure the `Incomplete` is always detected as a end of branch ? We can never have something like : 

```
( -266.79  -131.77   -26.13     
Incomplete       
```
or even:
```
 ( -266.79  -131.77   -26.13     0.92) 
 Incomplete
 ( -268.17  -130.62   -24.75     0.92)
```
before a end of split ?

Reply:
I understand that 'Incomplete' always ends the branch from my discussion with @arnaudon. We can additionally validate other cases and throw.

Reply:
Yes, it is to mark a branch in incomplete, so it must be at a leave branch. I think you can safely throw if not